cypress/e2e/parallel-4/pageskin.cy.ts

```diff
@@ -0,0 +1,75 @@
+import { breakpoints } from '../../fixtures/breakpoints';
+import { frontWithPageSkin } from '../../fixtures/pages';
+import { mockIntersectionObserver } from '../../lib/util';
+
+const gamUrl = 'https://securepubads.g.doubleclick.net/gampad/ads?**';
+
+const large = breakpoints.filter(
+	({ breakpoint }) => breakpoint === 'desktop' || breakpoint === 'wide',
+);
+
+const small = breakpoints.filter(
+	({ breakpoint }) => breakpoint === 'mobile' || breakpoint === 'tablet',
+);
+
+describe('pageskin on uk front', () => {
+	beforeEach(() => {
+		cy.useConsentedSession('pageskin-consented');
+	});
+
+	large.forEach((breakpoint) => {
+		it(`Test ${frontWithPageSkin.path} on ${breakpoint.breakpoint} should display the pageskin background and use single request mode`, () => {
+			cy.viewport(breakpoint.width, breakpoint.height);
+
+			cy.intercept(gamUrl, (req) => {
+				req.continue((res) => {
+					expect(res.headers).to.have.property('google-lineitem-id');
+					expect(
+						(res.headers['google-lineitem-id'] as string).split(',')
+							.length,
+					).to.be.greaterThan(1);
```

Comment:
Does this `expect` test that we're using `single request mode`. If so, why?

Reply:
Tom Duffet showed us the `google-lineitem-id` header during the incident, in single request mode it'll have all the ads on the page, in "normal" mode it'll just have a single lineitem.

Will clear it up in the code